x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -176,23 +193,22 @@ docker.elastic.co/observability/profiling-agent:${hostAgentVersion} /root/pf-hos
       ],
     },
     {
-      key: NoDataTabs.Deb,
+      key: AddDataTabs.Deb,
       title: i18n.translate('xpack.profiling.tabs.debTitle', {
         defaultMessage: 'DEB Package',
       }),
       steps: [
         {
-          title: i18n.translate('xpack.profiling.tabs.debDownloadPackageStep', {
-            defaultMessage:
-              'Open the URL below and download the right DEB package for your CPU architecture:',
+          title: i18n.translate('xpack.profiling.tabs.debConfigureRepoStep', {
+            defaultMessage: 'Configure the apt repository (requires root privileges):',
           }),
           content: (
-            <EuiLink
-              target="_blank"
-              href={`https://ela.st/pf-host-agent-linux-${hostAgentVersion}`}
-            >
-              {`https://ela.st/pf-host-agent-linux-${hostAgentVersion}`}
-            </EuiLink>
+            <EuiCodeBlock paddingSize="s" isCopyable>
+              {`wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+sudo apt-get install apt-transport-https
+echo "deb https://artifacts.elastic.co/packages/${majorVersion}.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-${majorVersion}.x.list
```

Comment:
The IDE is reporting a warning here, saying that this variable is used before intialization.
WDYT?

Reply:
That's because you just defined the variable but you did not initialise it with any value. But this message will be gone if you apply the changes I suggested here: https://github.com/elastic/kibana/pull/163884/files#r1294302786